kcc3/settings/settings.py

```diff
@@ -0,0 +1,41 @@
+from .base import *
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get(
+    'DJANGO_SECRET_KEY', '3on#7^-^=nxx0_npqt3r_gr6c#5))vdt!v5n876_vv78em9b!v')
```

Comment:
I know it's just the default value in case the user doesn't set it, but I think there should be more of a placeholder-type value like `PLACEHOLDER_KEY_CHANGE_ME` or sth. Seeing a key like that might prompt the user not to change it since it's "secure" already.